app/views/layouts/_page_header.html.erb

```diff
@@ -9,6 +9,7 @@
         signOutEndpoint: '<%=destroy_user_session_path%>',
         userName: '<%= current_user.name %>',
         userSignedIn: true,
+        allowedFeatures: JSON.parse('<%= raw escape_json(@current_user.allowed_feature_list) %>'),
         disableNavigation: <%= @disable_header_navigation %>,
       }, 'page_header', JSON.parse('<%= raw escape_json(request_context)%>'))
```

Comment:
Note the request context being passed here... so you should not need to pass allowedFeatures.